client/src/modules/roles/modal/rolesPermissions.html

```diff
@@ -45,7 +45,7 @@ <h4>
 <div class="modal-footer">
 
   <div pull-right>
-    <button class='btn btn-default' ng-click='RolesPermissionsCtrl.close()' translate>FORM.BUTTONS.CANCEL</button>
+    <span class='btn btn-default' ng-click='RolesPermissionsCtrl.close()' translate>FORM.BUTTONS.CANCEL</span>
```

Comment:
If you are going to click an element, it should really be either a `button` or and `a` element.  Why was this change necessary?